app/views/keywords/show.html.slim

```diff
@@ -0,0 +1,53 @@
+div class="d-flex flex-column align-items-center mt-4"
+
+  h1.display-3 == keyword.keyword
+
+  - if keyword.completed?
+    div class="d-flex mt-5 w-100 justify-content-center"
```

Comment:
just small thing, no need `div` here, please check for all `slim` file :)

Reply:
Fixed in bd96b0027c04fef6a667361757a3eeec2e229f7a